ospatch/system_linux.go

```diff
@@ -35,7 +35,7 @@ func disableAutoUpdates() {
 		if err != nil {
 			if eerr, ok := err.(*exec.ExitError); ok {
 				// Error code of 1 indicates disabled.
```

Comment:
Does the comment mean "was already disabled" or "was successfully disabled" or both?

Reply:
Is currently disabled, the above check is just running "is-enabled"